### PR TITLE
Staging resources updates

### PR DESCRIPTION
**Pull request by benjwi**

### CODE REVIEW

**Comment by benjwi** [comment]
**Mongodb**: 
- Standalone
- Resource limited
- persistenceStorage

Values **db.uri** update path

**MongoDb** express already low, [not configurable](https://github.com/w3f/mongo-express-chart/blob/master/charts/mongo-express/templates/deployment.yaml) 

**otv-backend** 
- Resource limited
- Storage

File: apps/1kv-backend/values.yaml [hunk 1]
```diff
@@ -13,13 +15,55 @@ kusama:
...
   mongoExpress:
     whitelistSourceIps: <path:vaults/k8s-community-secrets/items/otv-polkadot#shared-vpn-ip>
-    domain: w3f.community      
\ No newline at end of file
+    domain: w3f.community
+
+resources:
+  kusama:
+    db:
+      storageSize: "8Gi"
```
Comment (ironoa) [inline apps/1kv-backend/values.yaml:34]: kusama.db.storageSize

File: apps/1kv-backend/values.yaml [hunk 2]
```diff
@@ -13,13 +15,55 @@ kusama:
...
+    resources:
```
Comment (ironoa) [inline apps/1kv-backend/values.yaml:35]: kusama.db.resources

File: apps/1kv-backend/values.yaml [hunk 3]
```diff
@@ -13,13 +15,55 @@ kusama:
...
+      limits: 
+        cpu: "1200m"
+        memory: "2Gi"
+      requests: 
+        cpu: "1000m"
+        memory: "1Gi"
+    be:
+      storage: "20Gi"
```
Comment (ironoa) [inline apps/1kv-backend/values.yaml:62]: keep it consistent, either storage or storageSize

File: charts/otv-backend/templates/statefulset.yaml [hunk 4]
```diff
@@ -75,5 +75,5 @@ spec:
       accessModes: [ "ReadWriteOnce" ]
       resources:
         requests:
-          storage: 20Gi
+          storage: {{ .Values.storage }}
```
Comment (ironoa) [inline charts/otv-backend/templates/statefulset.yaml:78]: missing a be ?

File: apps/1kv-backend/values.yaml (hunk 2 above)
Comment (benjwi) [reply apps/1kv-backend/values.yaml]: ok

File: apps/1kv-backend/values.yaml (hunk 3 above)
Comment (benjwi) [reply apps/1kv-backend/values.yaml]: ok

File: apps/1kv-backend-staging/values.yaml [hunk 5]
```diff
@@ -1,25 +1,61 @@
...
+        cpu: "300m"
+        memory: "400Mi"
   be:
+    nominating: false
     secret:  <path:vaults/k8s-community-secrets/items/otv-polkadot#backend-secret>
     domain: polkadot-staging.w3f.community
     matrix:   
       room: "!mAeaxHqMUzRnTUhFUM:web3.foundation"
-      userId: "@1kv-stats:matrix.org"
+    userId: "@1kv-stats:matrix.org"
```
Comment (ironoa) [inline apps/1kv-backend-staging/values.yaml:50]: watch out, wrong identation for userId

**Pull request merged**